sql/functions/layer_country_labels.sql

```diff
@@ -11,8 +11,7 @@ AS $BODY$
     osm_id, 
     way, 
     name,
---    (hstore_to_json(extract_names(tags)))::text name_,
-    '' AS name_, 
```

Comment:
I guess this was just a placeholder select to allow the queries to run without the function right?

Reply:
Correct. I wanted to keep the function signature consistent with the ultimate goal. Also makes for a quick cutover like this. 😉